test/e2e/resource/create_test.go

```diff
@@ -0,0 +1,704 @@
+					return err
+				}
+
+				if _, err := c.Send(string(terminal.KeyArrowDown)); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("cluster"); err != nil {
+					return err
+				}
+
+				if _, err := c.Send(string(terminal.KeyEnter)); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Enter a value for url :"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine("https://1.1.1.1"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Enter a value for username :"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine("test-user"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Is the cluster secure"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("yes"); err != nil {
+					return err
+				}
+
+				if _, err := c.Send(string(terminal.KeyEnter)); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Which authentication technique you want to use"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("password"); err != nil {
+					return err
+				}
+
+				if _, err := c.Send(string(terminal.KeyArrowDown)); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("token"); err != nil {
+					return err
+				}
+
+				if _, err := c.Send(string(terminal.KeyEnter)); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("How do you want to set cluster credentials"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Passing plain text as parameters"); err != nil {
+					return err
+				}
+
+				if _, err := c.Send(string(terminal.KeyArrowDown)); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Using existing kubernetes secrets"); err != nil {
+					return err
+				}
+
+				if _, err := c.Send(string(terminal.KeyEnter)); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Secret Key for token :"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine("tokenkey"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Secret Name for token :"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine(secretName); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Secret Key for cadata :"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine("cadatakey"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Secret Name for cadata :"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine(secretName); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("New cluster resource \"my-cluster\" has been created"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectEOF(); err != nil {
+					return err
+				}
+
+				c.Close()
+				return nil
+			}})
+	})
+
+	t.Run("list single pipeline resource", func(t *testing.T) {
```

Comment:
I would change the name to something more descriptive about which PipelineResource.